ci/unit.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16.3
+    tag: 1.16.7
```

Comment:
if specific version is not strictly required then it would be better to use with latest image to avoid an ambiguity in dependency management.

Reply:
I think https://github.com/ONSdigital/dp/blob/main/standards/DEPENDENCY_UPGRADING.md#build-and-test-version-docker-image would disagree.
We want predictable builds on specific versions so that we can reliably recreate what is in the environment.